sdk/service/sdk.go

```diff
@@ -1,38 +1,64 @@
 package servicesdk
 
 import (
-	"fmt"
-
+	"github.com/cosmos/cosmos-sdk/codec"
+	"github.com/mesg-foundation/engine/cosmos"
 	"github.com/mesg-foundation/engine/hash"
 	"github.com/mesg-foundation/engine/protobuf/api"
 	"github.com/mesg-foundation/engine/service"
 )
 
-type sdk struct {
+type SDK struct {
+	cdc    *codec.Codec
+	client *cosmos.Client
 }
 
 // NewSDK returns the service sdk.
-func NewSDK() Service {
-	sdk := &sdk{}
+func NewSDK(cdc *codec.Codec, client *cosmos.Client) Service {
+	sdk := &SDK{
+		cdc:    cdc,
+		client: client,
+	}
 	return sdk
 }
 
 // Create creates a new service from definition.
-func (s *sdk) Create(req *api.CreateServiceRequest) (*service.Service, error) {
-	return nil, fmt.Errorf("create not implemented")
+func (s *SDK) Create(req *api.CreateServiceRequest) (*service.Service, error) {
+	// TODO: pass account by parameters
+	accountName, accountPassword := "bob", "pass"
+	accNumber, accSeq := uint64(0), uint64(0)
```

Comment:
There is no seq increment -  it won't work with multiple service, will it? the same is for delete method

Reply:
try to create a service and delete it. it's working. maybe the sequence needs to be set when doing multiple transaction in the same block?

Reply:
I think this works because create then delete will be 2 different type of transaction so the final hash will anyway be different but if you do 2 similar transactions (like re-create the service after) then it will fail because the sequence will be 0 and it will generate exactly the same tx.

This `accSeq` or `accNumber` (not sure what's the difference) should be incremented. This is a guess but it would make sense and that could be the reason why you cannot recreate the same service.

When I manually increment the seq and recreate the service (after deleting it) everything works fine. Same if I don't delete the service the transaction goes through and I have an error that the service already exists.